.github/workflows/ci.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           # installs.
           rm -f iree-install/bin/clang*
           rm -f iree-install/bin/llvm-link*
-          tar cf iree-dist.tar -C iree-install .
+          tar cf iree-dist.tar -C iree-install . -C ../iree-build tools/iree-e2e-matmul-test
```

Comment:
Shouldn't you make sure this is added to the install in the cmake code?

Reply:
It was decided that we dont want it in install
https://discord.com/channels/689900678990135345/1169307503633383664/1208082521720488016

Reply:
We chatted about this over on [IREE's Discord server here](https://discord.com/channels/689900678990135345/1169307503633383664/1208069599178924082). `iree-e2e-matmul-test` is a dev tool that should not be included in the user install. The options here were to either include it in the archive here or [re]build it from source in the next job.